XamarinCommunityToolkit/Behaviors/Animations/AnimationTypes/AnimationBase.shared.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors.Animations
+{
+    public abstract class AnimationBase : BindableObject
+    {
+        public static readonly BindableProperty DurationProperty =
+            BindableProperty.Create(nameof(Duration), typeof(uint), typeof(AnimationBase), uint.MinValue,
```

Comment:
The default could also just be `default(uint)` I would assume?

Reply:
Yes! Done!